### DISCUSSION TITLE
Web-based editor: handle back button more gracefully

### DISCUSSION THREAD

**Discussion by kaycebasques** [community/community#5456] [Other Feature Feedback, Questions, & Ideas]
Looks nice! This was a bit annoying, however:

1. Open the web-based editor by pressing `.` on a source code page on `github.com`
2. Use browser's **back** button to attempt to go back to the `github.com` page

Once you're in the web-based editor, you're kinda trapped. Can't go back to the page you were viewing on `github.com`. Well, when I clicked like 3 or 4 times, it kinda worked.

**Comment by SamB** [comment]
I'm on mobile here, my experience was basically the opposite of that:

1. Open the editor by editing the URL from github.com to github.dev
2. Attempt to go back a step in the "getting started" wizard thing (which is gone now, so I'm not sure of the exact name) using the back button on my phone's navbar
3. Ended up back at github.com instead of going back one step in the wizard

I'm not saying that OP's expectation was wrong, but how come they didn't see the same thing as I did?

**Comment by bamurtaugh** [comment]
Thanks for the discussion here. I believe pressing the back button just once should redirect to the previous page (whether it was github.com or something else). @kaycebasques are you able to repro this regularly? If so, could you share the repo you opened and what actions you performed while in github.dev before attempting to go back?
